Sources/ReleaseServer/ReleaseServer.WebApi/Controllers/ReleaseArtifactController.cs

```diff
@@ -178,21 +277,28 @@ public async Task<FileStreamResult> Backup()
             };
         }
 
+        /// <summary>
+        /// Restores the uploaded backup file.
+        /// </summary>
+        /// <param name="backupFile"></param>
+        /// <returns></returns>
+        /// <response code="200">The restore process was successful.</response>
+        /// <response code="401">The user is not authorized (wrong credentials or missing auth header).</response>
         [HttpPut("restore")]
-        public async Task<IActionResult> Restore()
+        public async Task<IActionResult> Restore([Required] IFormFile backupFile)
         {
-            var payload = Request.Form.Files.FirstOrDefault();
-            
-            if (payload == null)
+            if (backupFile == null)
                 return BadRequest();
```

Comment:
Please add this 400 error code in the XML documentation.

Reply:
fixed

Reply:
see https://github.com/Traeger-GmbH/release-server/pull/50/commits/89c4e62a1637ac6d990d290b01c6e126d5ef6900